lib/salus/report.rb

```diff
@@ -229,9 +231,28 @@ def publish_report(directive)
       end
     end
 
+    def satisfies_filter?(directive)
+      return true if @report_filter == 'all'
+      return false if @report_filter == 'none'
+
+      # Split the filter string only by the first colon
+      recovered_values = @report_filter.split(':', 2)
+      filter_key = recovered_values[0]
+      filter_value = recovered_values[1]
+
+      if filter_key.nil? || filter_key == '' || filter_value.nil? || filter_value == ''
```

Comment:
This code is fine, you don't need to change it. But if you ever want to make code like this shorter, you could use
```ruby
if filter_key.to_s == '' || filter_value.to_s == ''
```